### PR TITLE
Update test_reflective_loader, fix analyzer warnings&errors

### PR DESCRIPTION
**Pull request by MichaelRFairhurst**
Cannot use the version we were using before. And I'm not sure how these analysis errors made it through travis, but they are bad too

### CODE REVIEW

File: analyzer_plugin/test/angular_driver_test.dart [hunk 1]
```diff
@@ -2487,7 +2488,7 @@ class ComponentA {
 }
 ''';
     Source dartSource = newSource('/weird.dart', code);
-    Soruce htmlSource =
+    Source htmlSource =
```
Comment (liuming0) [inline analyzer_plugin/test/angular_driver_test.dart]: User var instead of Source.

File: server_plugin/test/analysis_test.dart [hunk 2]
```diff
@@ -25,10 +25,11 @@ import 'package:unittest/unittest.dart';
 import 'mock_sdk.dart';
 
 main() {
-  groupSep = ' | ';
-  //defineReflectiveTests(AngularNavigationContributorTest);
-  //defineReflectiveTests(AngularOccurrencesContributorTest);
-  defineReflectiveTests(EmptyTest);
+  defineReflectiveSuite(() {
+    //defineReflectiveTests(AngularNavigationContributorTest);
+    //defineReflectiveTests(AngularOccurrencesContributorTest);
```
Comment (liuming0) [inline server_plugin/test/analysis_test.dart]: Remove the //..

File: analyzer_plugin/test/angular_driver_test.dart (hunk 1 above)
Comment (MichaelRFairhurst) [inline analyzer_plugin/test/angular_driver_test.dart]: good call

File: server_plugin/test/analysis_test.dart (hunk 2 above)
Comment (MichaelRFairhurst) [inline server_plugin/test/analysis_test.dart]: these actually have to be disabled :(

unless, is there a better way to skip them with whatever the heck test version we're using?

**Pull request merged**